test/tables/sortable-table.test.js

```diff
@@ -476,4 +476,15 @@ describe('SortableTable', () => {
     )
     expect(screen.getByText('My Table')).toHaveClass('custom-caption')
   })
+
+  test('Column with custom className is propagated to TableHeader', () => {
+    render(
+      <SortableTable data={tableData}>
+        <Column name="name" className="foo" />
+      </SortableTable>
+    )
+
+    const header = screen.getByText('Name').closest('th')
```

Comment:
Why do you need `closest` here? When I printed out the value for `screen.getByText('Name')`, it was the header itself 🤔